### ISSUE TITLE
WiFi Connect not asking for password

### ISSUE THREAD

**Issue by fmarzocca** [wifiphisher/wifiphisher#1130]
I am running apt version of wifiphisher on Kali Linux (1.4GIT).
Once started, selected the AP and the mode (Network Manager Connect) I tried to connect to the rogue AP, but I only get the error page (see screenshot) and never get asked for the password to connect. 
![Schermata 2019-05-11 alle 11 52 21](https://user-images.githubusercontent.com/8530030/57568285-67f00900-73e5-11e9-86ba-f5ff9edffb1f.png)

**Comment by sophron** [comment]
Yes, I don't think that JS will run on the captive portal window. 

Can you try opening the browser and hitting `neverssl.com`? Do you now see the window?
